apps/web/modules/settings/billing/components/BillingCredits.tsx

```diff
@@ -132,117 +169,154 @@ export default function BillingCredits() {
 
   return (
-    <div className="border-subtle mt-8 space-y-6 rounded-lg border px-6 py-6 pb-6 text-sm sm:space-y-8">
-      <div>
-        <h2 className="text-base font-semibold">{t("credits")}</h2>
-        <ServerTrans
-          t={t}
-          i18nKey="view_and_manage_credits_description"
-          components={[
-            <Link
-              key="Credit System"
-              className="underline underline-offset-2"
-              target="_blank"
-              href="https://cal.com/help/billing-and-usage/messaging-credits">
-              Learn more
-            </Link>,
-          ]}
-        />
-        <div className="-mx-6 mt-6">
-          <hr className="border-subtle" />
+    <>
+      <div className="bg-muted border-muted mt-5 rounded-xl border p-1">
+        <div className="flex flex-col gap-1 px-4 py-5">
+          <h2 className="text-default text-base font-semibold leading-none">{t("credits")}</h2>
+          <p className="text-subtle text-sm font-medium leading-tight">{t("view_and_manage_credits")}</p>
         </div>
-        <div className="mt-6">
-          {creditsData.credits.totalMonthlyCredits > 0 ? (
-            <div className="mb-4">
-              <Label>{t("monthly_credits")}</Label>
-              <ProgressBar
-                color="green"
-                percentageValue={teamCreditsPercentageUsed}
-                label={`${Math.max(0, Math.round(teamCreditsPercentageUsed))}%`}
-              />
-              <div className="text-subtle">
-                <div>
-                  {t("total_credits", {
-                    totalCredits: creditsData.credits.totalMonthlyCredits,
-                  })}
+        <div className="bg-default border-muted flex w-full rounded-[10px] border px-5 py-4">
+          <div className="w-full">
+            {totalCredits > 0 ? (
+              <>
+                <div className="mb-4">
+                  <CreditRow
+                    label={t("monthly_credits")}
+                    value={creditsData.credits.totalMonthlyCredits ?? 0}
+                    isBold={true}
+                    underline="dashed"
+                  />
+                  <CreditRow label={t("credits_used")} value={totalUsed} underline="solid" />
+                  <CreditRow
+                    label={t("total_credits_remaining")}
+                    value={creditsData.credits.totalRemainingCreditsForMonth}
+                  />
+                  <div className="mt-4">
+                    <ProgressBar color="green" percentageValue={100 - teamCreditsPercentageUsed} />
+                  </div>
+                  {/*750 credits per tip*/}
+                  <div className="mt-4 flex flex-1 items-center justify-between">
+                    <p className="text-subtle text-sm font-medium leading-tight">
+                      {orgSlug ? t("credits_per_tip_org") : t("credits_per_tip_teams")}
+                    </p>
+                    <Button
```

Comment:
I think this should open the dialog and not redirect to `/members`
<img width="623" height="438" alt="Screenshot 2025-09-19 at 10 50 36 AM" src="https://github.com/user-attachments/assets/46cacc38-250a-4983-ad83-9943e868eb7b" />